src/retriever/retriever.js

```diff
@@ -1,9 +1,15 @@
+import { AxiosHeaders } from "axios";
 import { CONFIG } from "../config.js";
 import { HTML_TYPES, Requests } from "../types.js";
 import { Session, getPageText } from "./session.js";
 import { urls } from "./url.js";
 import { parseHttpEquivHeaders } from "./utils.js";
 
+const STANDARD_HEADERS = [
+  "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
+];
+const ROBOTS_HEADERS = ["Accept: text/plain,*/*;q=0.8"];
```

Comment:
Where do these come from?

Reply:
`STANDARD_HEADERS` is derived from what the browser is sending, sans prioritizing images over catch-all `*/*`. `ROBOT_HEADERS` is derived from the specs, i.e. it is a plain-text file.